bin/fetch_xml_getty_records

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# fetch_xml_getty_records - download the most recent dump of xml files from the getty research portal
+# requires wget and xq (on macos: `brew install wget python-yq`)
+
+CUR=`dirname $0`
+mkdir -p $CUR/../tmp
```

Comment:
We should probably just add a `/tmp/.keep` file to make sure this dir exists, and add it to `.gitignore`

Reply:
done.